app/models/comment.rb

```diff
@@ -0,0 +1,6 @@
+class Comment < ActiveRecord::Base
+  belongs_to :commentable, polymorphic: true
+  belongs_to :user
+
+  validates :comment_body, :user_id, { presence: true }
```

Comment:
It's better to validate user rather than user_id - that way rails will check the user exists as well as that the column is not null.